src/pages/Home/Home.module.scss

```diff
@@ -0,0 +1,9 @@
+.container{
+    padding-left: 20px;
+    padding-right: 20px;
+    background-color: var(--bg_body);
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    flex: 1;}
```

Comment:
You acutely don't need to use any css file, all styles should go with MUI css in JS "SX".
see the above comment